static/src/js/components/AdminRetrievals/__tests__/AdminRetrievals.test.jsx

```diff
@@ -1,40 +1,36 @@
-import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
+import { screen } from '@testing-library/react'
 
+import setupTest from '../../../../../../jestConfigs/setupTest'
 import { AdminRetrievals } from '../AdminRetrievals'
-import { AdminPage } from '../../AdminPage/AdminPage'
-import { AdminRetrievalsList } from '../AdminRetrievalsList'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup() {
-  const props = {
+const setup = setupTest({
+  Component: AdminRetrievals,
+  withRouter: true,
+  defaultProps: {
     historyPush: jest.fn(),
     onAdminViewRetrieval: jest.fn(),
+    onFetchAdminRetrievals: jest.fn(),
     onUpdateAdminRetrievalsSortKey: jest.fn(),
     onUpdateAdminRetrievalsPageNum: jest.fn(),
     retrievals: {
       allIds: [],
       byId: {},
-      pagination: {},
+      pagination: {
+        pageNum: 1,
+        pageSize: 20,
+        totalResults: 0
+      },
       sortKey: ''
     }
   }
-
-  const enzymeWrapper = shallow(<AdminRetrievals {...props} />)
-
-  return {
-    enzymeWrapper,
-    props
-  }
-}
+})
 
 describe('AdminRetrievals component', () => {
   test('renders itself correctly', () => {
```

Comment:
This test isn't really testing anything useful. It is showing that the `AdminRetrievalsForm` component is rendered, but it isn't testing anything with `AdminRetrievalsList`.

If you mock the child components, and update the props to include a retrieval, then you can assert that the child components are called with the right props, which better tests what this component is actually doing